.github/workflows/ci.yml

```diff
@@ -2,59 +2,57 @@ name: CI
 
 on:
   push:
-    branches: [ master ]
+    branches: [master]
 
   pull_request:
-    branches: [ master ]
+    branches: [master]
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
 
     strategy:
       matrix:
-        node-version: [12.x, 14.x]
+        node-version: [12.x, 14.x, 18.x]
```

Comment:
I think we can drop 12.x because it has reached the end of life. We can add 16.x instead.